client/server/middleware/assets.js

```diff
@@ -58,9 +58,11 @@ export default () => {
 				return EMPTY_ASSETS;
 			}
 
-			const allTheFiles = chunk.files.concat(
-				flatten( chunk.siblings.map( ( sibling ) => getChunkById( assets, sibling ).files ) )
-			);
+			const allTheFiles = chunk.chunks
+				? flatten( chunk.chunks.map( ( chunkId ) => getChunkById( assets, chunkId ).files ) )
+				: chunk.files.concat(
+						flatten( chunk.siblings.map( ( sibling ) => getChunkById( assets, sibling ).files ) )
+				  );
```

Comment:
The new code is good, but to keep the code readable we should remove the old code. The assets writer plugin doesn't need to output `siblings` any more, they are not used. And `getFilesForChunk` should work only with `chunk.chunks`. `getFilesForChunk` is called only for chunks that are the main chunks of a chunk group, and the `chunk.chunks` should always be there. If it's not, it's a reason to warn about an unknown chunk group and return empty assets.

Reply:
Is [bin/loader-stats.js](https://github.com/Automattic/wp-calypso/blob/999103d342eba409b228944c1bdcf21c7b0c3906/bin/loader-stats.js#L36) still in use?

Reply:
I think you can safely delete it, it was relevant many years ago. Its job is now done by the iscalypsofastyet.com server and the GitHub comments that bot is posting.

But even if you wanted to keep it should be easy to update it to use `chunk.chunks` instead of `chunk.siblings`. The `getChunkAndSiblings` just gets the list of chunks in a chunk group, a task we are already familiar with.